cmd/podman/containers/run.go

```diff
@@ -76,7 +76,20 @@ func init() {
 		Mode:    []entities.EngineMode{entities.ABIMode, entities.TunnelMode},
 		Command: runCommand,
 	})
+	addDetach := func(flags *pflag.FlagSet) {
+		flags.BoolVarP(
+			&cliVals.Detach,
+			"detach", "d", false,
+			"Run container in background and print container ID",
+		)
+		flags.StringVar(
+			&cliVals.DetachKeys,
+			"detach-keys", containerConfig.DetachKeys(),
+			"Override the key sequence for detaching a container. Format is a single character `[a-Z]` or a comma separated sequence of `ctrl-<value>`, where `<value>` is one of: `a-cf`, `@`, `^`, `[`, `\\`, `]`, `^` or `_`",
+		)
+	}
 	flags := runCommand.Flags()
```

Comment:
Why the extra function? This could be added to the `runFlags` function (line 58).

Reply:
Thanks, I was rushing to get this done.  This is a much cleaner solution.